### ISSUE TITLE
Back button & Text reflow both broken on LineageOS 14.1

### ISSUE THREAD

**Issue by fake-fur** [anthonycr/Lightning-Browser#662]
hi

thanks for all the hard work - really appreciated 

the last update (4.5.1 i think) seems to have broken the back button behaviour, and text reflow is no longer working at all

**Comment by jrbray1** [comment]
The back button is now going back 2 pages on sites like the bbc news site. Its making it hard to use the browser

**Comment by Rexelpitlum** [comment]
Broken back button is not restricted to LineageOS.
I have a Galaxy S5 which still runs the Samsung Standard ROM.
Here the back button not only goes back 2 pages instead of one, but goes back *all* pages in tab history belonging to the same base URL.
This makes Lightning nearly unusable at the moment.

**Comment by Micha-Btz** [comment]
#604 seems to be also the root off my problem.

**Comment by fake-fur** [comment]
yes i had noticed the same or similar behaviour on the lineageos default gello(?) browser, and did start to wonder if somehow the underlying browser control had been borked by google, hard though that was to believe

good find on the confirmation - thanks

**Comment by Micha-Btz** [comment]
LineageOS has updated the webview version to 63.0.3239.111 
https://github.com/LineageOS/android_external_chromium-webview/commit/ea18916ffc0b2e03edc337d301e29cb9197113dc
and I have also tested with the beta version 64.0.3282.52 but both doesn't work well. By all versions the back button is not worked as expected. Sometimes it goes back more then one page, sometimes more then 10 pages or even to a page that simply not exist. 
I go back to webview 60.0.3112.78 and this version is working correctly. 
Maybe lightning needs adjustment to the newer version. 
Any one else with some experience?

**Comment by casta** [comment]
63 will not be fixed. The current 64 beta from 2017-12-14 is NOT fixed. But the next beta will be. Info taken from the chromium bug report. No need to fix anything in lightning. The fix is in webview.

**Comment by casta** [comment]
Confirmed that the beta from yesterday (64.0.3282.73) on playstore fixes the issue
